README.md

```diff
@@ -5,13 +5,12 @@
 1. [Wprowadzenie](#wprowadzenie)
 2. [Grupy ryzyka zarażenia Sars-Cov-2](#grupy-ryzyka-zarażenia-sars-cov-2)
 3. [Anonimowość i bezpieczeństwo](#anonimowość-i-bezpieczeństwo)
-4. [Grupy ryzyka zarażenia Sars-Cov-2](#grupy-ryzyka-zarażenia-sars-cov-2)
-5. [Dalsze założenia](#dalsze-założenia)
-6. [Zakres 2.0](#zakres-20)
-7. [Zakres 3.0](#zakres-30)
-9. [Zakres 3.2](#zakres-32)
-10. [Najczęściej zadawane pytania](FAQ.md)
-11. [Chcę pomóc, zgłosić błąd, mam pomysł](CONTRIBUTING.md)
+4. [Dalsze założenia](#dalsze-założenia)
+5. [Zakres 2.0](#zakres-20)
+6. [Zakres 3.0](#zakres-30)
+7. [Zakres 3.2](#zakres-32)
+8. [Najczęściej zadawane pytania](FAQ.md)
+9. [Chcę pomóc, zgłosić błąd, mam pomysł](CONTRIBUTING.md)
```

Comment:
A wygląda jakbyś usunął i napisał to samo. 
![image](https://user-images.githubusercontent.com/64913064/82070016-734ed200-96d4-11ea-8645-a5b458c6cb00.png)

Reply:
A nie teraz już widzę.
![image](https://user-images.githubusercontent.com/64913064/82070140-98dbdb80-96d4-11ea-8a94-be742168b52d.png)

Reply:
Pominięto jeszcze w oryginale punkt 8, zapomniałem o tym wspomnieć w PR.